src/Components/Footer.js

```diff
@@ -1,63 +1,69 @@
-import React from 'react'
```

Comment:
Nit: Would you mind running the linter to add missing semi colons and what not just to avoid merge conflicts from linting issues?

Reply:
Ah Nevermind, I see you actually did that and I was looking at the old code.  Good job haha!!